include/rosidl_dynamic_typesupport/dynamic_message_type_support_struct.h

```diff
@@ -89,21 +90,53 @@ typedef struct rosidl_dynamic_message_type_support_impl_s
  */
 ROSIDL_DYNAMIC_TYPESUPPORT_PUBLIC
 rcutils_ret_t
-rosidl_dynamic_message_type_support_handle_create(
+rosidl_dynamic_message_type_support_handle_init(
   rosidl_dynamic_typesupport_serialization_support_t * serialization_support,
   const rosidl_type_hash_t * type_hash,
   const rosidl_runtime_c__type_description__TypeDescription * type_description,
   const rosidl_runtime_c__type_description__TypeSource__Sequence * type_description_sources,
-  rosidl_message_type_support_t ** ts);  // OUT
+  rcutils_allocator_t * allocator,
+  rosidl_message_type_support_t * ts);  // OUT
 
-/// Destroy a rosidl_message_type_support_t obtained with
-/// `rosidl_dynamic_message_type_support_handle_create()`, which has dynamically allocated members
+/// Finalize a rosidl_message_type_support_t obtained with
+/// `rosidl_dynamic_message_type_support_handle_init()`, which has dynamically allocated members
 ///
 /// NOTE: Using this on a statically allocated typesupport will cause undefined behavior!
 ///       (Static memory will get freed in that case.)
 ROSIDL_DYNAMIC_TYPESUPPORT_PUBLIC
 rcutils_ret_t
-rosidl_dynamic_message_type_support_handle_destroy(rosidl_message_type_support_t * ts);
+rosidl_dynamic_message_type_support_handle_fini(rosidl_message_type_support_t * ts);
+
+/// Initialized a `rosidl_dynamic_message_type_support_impl_t` with bound message description
```

Comment:
What does `bound` mean in this context?

Reply:
I think I should rephrase it to `encapsulated`